Terratest/vpn/main.tf

```diff
@@ -0,0 +1,225 @@
+locals {

```

Comment:
terratest must be adjusted to run only on one of the stages (either de test or nl test) as this test uses both regions anyway and must not be tested on two test tasks